flink-runtime/src/main/java/org/apache/flink/runtime/client/JobClient.java

```diff
@@ -123,12 +122,12 @@ public static InetSocketAddress getJobManagerAddress(Configuration config) throw
 	 * @throws org.apache.flink.runtime.client.JobExecutionException Thrown if the job
 	 *                                                               execution fails.
 	 */
-	public static SerializedJobExecutionResult submitJobAndWait(
+	public static JobExecutionResult submitJobAndWait(
 			ActorSystem actorSystem,
 			ActorGateway jobManagerGateway,
 			JobGraph jobGraph,
 			FiniteDuration timeout,
-			boolean sysoutLogUpdates) throws JobExecutionException {
+			boolean sysoutLogUpdates, ClassLoader usercodeClassloader) throws JobExecutionException {
```

Comment:
Style: Why appending the new parameter to the last line when all the other parameters are written in a new line?